department-of-veterans-affairs/caseflow
# JHoang/APPEALS-23883

Resolves [23883](https://jira.devops.va.gov/browse/APPEALS-23883)

# Description
Add a call to special_issues_controller.rb#create when blue water or burn pit is checked during the AMA decision review process.

## Acceptance Criteria

## Testing Plan
1. Login as a Judge or Attorney with cases under review (AMA)
2. Navigate to the case details page for a case under review with issues.
3. Under the Actions dropdown, click on Ready for Dispatch or Decision Ready for Review
4. Add decisions to each issue and ensure at least one issue has Blue Water and or Burn Pit
5. Click Continue
6. Ensure POST request returns response code of 200 indicating the update has been successful.
<img width="1722" alt="Screenshot 2023-06-12 at 11 30 04 AM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/111081469/9a6c0df6-33b6-4158-8c6b-382c4fb2ed95">


# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
